README.md

```diff
@@ -95,6 +95,7 @@
   We integrate with common code editors:
 
   - [Alejandra extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=kamadorueda.alejandra)
+  - [doom-emacs configuration](https://github.com/hlissner/doom-emacs): `(set-formatter! 'alejandra  "alejandra --quiet" :modes '(nix-mode))`
```

Comment:
This hopefully encourages users to add more editors :)

Reply:
Absolutely! I wish I were more familiar with vim and emacs, but all I know to do is `nix run nixpkgs#vim` and `nix run nixpkgs#emacs` ✌🏻 

Thank you for your help